overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -84625,7 +84631,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "You\u0027re hot then you\u0027re cold",
-          "desc:8": "Place some Lava in the Aether! It\u0027ll immediately freeze into Aerogel which you can mine up...\n\n§6Note:§r Later you\u0027ll need to automate this. Currently there is a bug with Aether that prevents you from automating it with Dispensers or Fluid Placers, as the Lava doesn\u0027t turn into Aerogel when placed with them. Instead you\u0027ll need to use a Mechanical User, available in Chapter 13. (You won\u0027t need to automate it any earlier though.)\n\n\u0027Cause you\u0027re hot then you\u0027re cold,\nYou\u0027re yes then you\u0027re no,\nYou\u0027re in then you\u0027re out,\nYou\u0027re up then you\u0027re down,\nYou\u0027re wrong when it\u0027s right,\nIt\u0027s black and it\u0027s white,\nWe fight, we break up,\nTurn Lava to Ice!"
+          "desc:8": "Place some Lava in the Aether! It\u0027ll immediately freeze into Aerogel which you can mine up...\n\n§6Note:§r Later you\u0027ll need to automate this. You can use a mechanical user later on to automate this or any fluid placer to automate it now.\n\n\u0027Cause you\u0027re hot then you\u0027re cold,\nYou\u0027re yes then you\u0027re no,\nYou\u0027re in then you\u0027re out,\nYou\u0027re up then you\u0027re down,\nYou\u0027re wrong when it\u0027s right,\nIt\u0027s black and it\u0027s white,\nWe fight, we break up,\nTurn Lava to Ice!"
```

Comment:
any dispenser or fluid placer currently works (although this may change soon on some servers due to some technical issues), this should update to indicate as such